gfx/NeoDays/pngs_large_20x20/monsters/mon_zombie_hulk.json

```diff
@@ -0,0 +1,7 @@
+{
+  "id": [
+    "mon_zombie_hulk",
+    "mon_zombie_hulk_pk"
```

Comment:
```suggestion
```
this and thus the whole json can go away, mon_zombie_hulk_pk is a monster from a very old mod that doesn't ship with the game

Reply:
Removed json; I'll keep that information in mind for other monsters